mono/metadata/remoting.c

```diff
@@ -108,6 +108,9 @@ register_icall (gpointer func, const char *name, const char *sigstr, gboolean sa
 	mono_register_jit_icall (func, name, sig, save);
 }
 
+#define register_icall(func, name, sigstr, save) \
```

Comment:
Remove that.

Reply:
Understood. it is a way to cut a lot of line damage. I assume you understand that and will proceed with the alternative.

Reply:
About 12 lines of "damage" here.